zerver/tests/test_invite.py

```diff
@@ -3118,6 +3449,95 @@ def test_create_multiuse_link_with_specified_user_groups_api_call(self) -> None:
             {SystemGroups.MEMBERS, SystemGroups.FULL_MEMBERS},
         )
 
+    def test_edit_multiuse_invite_without_permission_to_subscribe_others(self) -> None:
+        realm = get_realm("zulip")
+        hamlet = self.example_user("hamlet")
+        iago = self.example_user("iago")
+        cordelia = self.example_user("cordelia")
+
+        stream_names = ["Denmark", "Scotland"]
+        streams = [get_stream(stream_name, realm) for stream_name in stream_names]
+        stream_ids = [stream.id for stream in streams]
+
+        multiuse_invite = MultiuseInvite.objects.create(
+            referred_by=self.example_user("iago"),
+            realm=realm,
+        )
+        multiuse_invite.streams.set(stream_ids)
+
+        def check_path_multiuse_invite_with_subscribe_permission(
+            user: UserProfile,
+            stream_ids: list[int],
+            allow_fail: bool = False,
+            *,
+            error_message: str | None = None,
```

Comment:
There is no need to pass `error_message` here if error message is same for all cases. Just having `allow_fail` parameter is enough.